src/mask_gen_utils/mask_refine.py

```diff
@@ -84,26 +91,62 @@ def _load_voxel_db(self) -> pd.DataFrame:
         if not self.voxel_inspection_results_db_path.exists():
             raise FileNotFoundError(f"Voxel inspection results database (csv) not found at {self.voxel_inspection_results_db_path}")
         return pd.read_csv(self.voxel_inspection_results_db_path)
-                
-    def _map_voxel_tags(self) -> Dict[str, str]:
-        """
-        Maps voxel inspection tags to canonical tags defined in the configuration.
+
+    def _get_df_needing_refinement(self) -> pd.DataFrame:
```

Comment:
@mkutu 
Here:

 the filtering of df for good tags is being done in a different step: _get_df_needing_refinement